app/controllers/admin/courts_controller.rb

```diff
@@ -129,5 +131,13 @@ def area_of_law_by_id_or_name
       return area_of_law if area_of_law.present?
       AreaOfLaw.where(name: AreaOfLaw::Name::CHILDREN).first
     end
+
+    def process_update_exception(e)
+      if e.message.include?('postcode_court')
```

Comment:
@garytaylor I'm trying to catch the exception that happens because validation on associated model fails. I didn't find a way how to catch that in a model. I'm happy to do it if you know how.